js/user.js

```diff
@@ -19,7 +19,8 @@ const setUserGreeting = (username, firstName, userProfilePicture) => {
     const userImgURL = userProfilePicture;
 
     userImgEl.forEach((element) => {
-      element.src = userImgURL;
+      if (!!userImgURL) element.src = userImgURL;
```

Comment:
Why do `!!userImgURL`? simply `userImgURL` will work in the if-case, its readable and understandable

Reply:
also, use proper `if(){} else{}` syntax its also easy to understand.

Reply:
Ok will do that